pkg/manager/manager_test.go

```diff
@@ -500,7 +501,7 @@ var _ = Describe("manger.Manager", func() {
 			It("should use the specified ResourceLock", func() {
 				m, err := New(cfg, Options{
 					LeaderElection:             true,
-					LeaderElectionResourceLock: resourcelock.ConfigMapsLeasesResourceLock,
+					LeaderElectionResourceLock: resourcelock.LeasesResourceLock,
```

Comment:
Upstream now only supports the Leases lock: https://github.com/kubernetes/client-go/blob/master/tools/leaderelection/resourcelock/interface.go#L166-L189

@alvaroaleman @vincepri I wonder if we should just drop the LeaderElectionResourceLock field. Doesn't make much sense to keep it if it can only be set to `LeasesResourceLock`. WDYT?